mypy.ini

```diff
@@ -596,6 +596,21 @@ ignore_errors = True
 [mypy-scipy.ndimage.morphology]
 ignore_errors = True
```

Comment:
I guess it will be okay to remove these now. Right?

Reply:
I'm guessing an underscore should be added (`._morphology`) or Mypy will still be unhappy.